index.html

```diff
@@ -167,6 +167,21 @@ <h5 class="display-3 text-center project-stat">100+</h5>
             </div>
             <br/>
             <div class="card-deck justify-content-center col-md-12">
+                <a href="irc.html" class="card col-md-4 shadow project-card">
```

Comment:
In every card use `col-lg-4` instead of `col-md-4`

Reply:
✅